docroot/themes/contrib/civictheme/civictheme_library/components/04-templates/page/page.twig

```diff
@@ -15,7 +15,7 @@
 {% set theme_class = 'civictheme-theme-%s'|format(theme|default('light')) %}
 {% set modifier_class = '%s %s'|format(theme_class, modifier_class|default('')) %}
 
-<div class="layout-container {{ modifier_class }}" {% if attributes %}{{ attributes|raw }}{% endif %}>
+<div class="{{ modifier_class }}" {% if attributes %}{{ attributes|raw }}{% endif %}>
```

Comment:
layout-container class must have been come when base templates were copied across, This is used in core Drupal themes to handle layout. 
But in reference to civic theme it has no use. Non relevant class. 
Removed it from civic theme.